gold-cc-cvc-input.html

```diff
@@ -67,6 +67,10 @@
         margin-left: 10px;
       }
 
+      iron-icon[hidden] {
```

Comment:
Is this only meant to target `iron-icon`? There's a label in there using the `hidden` attribute too.